apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/models/Sgv2ColumnDefinition.java

```diff
@@ -28,6 +29,7 @@ public Sgv2ColumnDefinition(
       example = "emailaddress",
       required = true,
       description = "Name for the column, which must be unique.")
+  @NotBlank(message = "columnDefinition.name must be provided")
```

Comment:
Is this working when it's placed on the method?

Reply:
Ok looks like there is no test covering missing/empty column name -- will add one.

Reply:
Added a test: column name validation does get invoked. Verified manually by commenting out annotation and then failure comes from gRPC side.

It actually makes sense that get method annotation is also used if field name is used -- Bean definition originally only considered accessors and so Field usage is sort of add-on.
But there's some room for error as validation (by Bean Validation impl) may use slightly different rules than databinding (Jackson etc).
There is also some overhead as validation impl has to traverse the object separate from deserialization: that's when get method is actually used during deserialization (which for databinding would only be used for serialization).